Data/Sys/GameSettings/RNO.ini

```diff
@@ -17,3 +18,5 @@
 [Video_Hacks]
 EFBToTextureEnable = False
 ImmediateXFBEnable = False
+# Multiple choice options displays some graphic corruption
+# DeferEFBCopies = False
```

Comment:
The "#" should be removed

Reply:
I did it on purpose since it hurts performance a good deal but this issue is minor and only happens for brief moments a few times during the multiple choice selections in the game

Reply:
Wouldn't it be good to mention that in the comment? Nobody would know the reasoning for the # otherwise.